src/types/file/index.ts

```diff
@@ -58,3 +58,7 @@ export interface TrashRequestItems {
     fileID: number;
     collectionID: number;
 }
+
+export interface DeleteFilesRequest {
```

Comment:
Do we need this extra interface, if you are just internally using it in the `FileService`?

Reply:
no, not really that necessary , as the `request data`  is simply a array

Reply:
added it as the request body was passed from between functions